tests/simple.cpp

```diff
@@ -94,11 +94,13 @@ TEST_CASE_METHOD(SimpleFixture, "Send/Recv Test", "[simple]")
 
   // atleast one message needs to be >2GB to test for the case of multi-piece
   // messages
-  std::size_t sizes[11] = { 288, 128, 64, 32, 64, 32, 32, 64, 32, 32 };
+  static const std::size_t sizes[] = { 288, 128, 64, 32, 64, 32, 32, 64, 32, 32 };
```

Comment:
Why not make it `std::array<size_t>` or even `std::vector<size_t>`? Then `sizes.size()` would work below.